slick/src/main/scala/akka/stream/alpakka/slick/javadsl/Slick.scala

```diff
@@ -64,6 +68,23 @@ object Slick {
   ): Flow[T, java.lang.Integer, NotUsed] =
     flow(session, 1, toStatement)
 
+  /**
+   * Java API: creates a Flow that takes a stream of elements of
+   *           type T, transforms each element to a SQL statement
+   *           using the specified function, and then executes
+   *           those statements against the specified Slick database.
+   *
+   * @param session The database session to use.
+   * @param toStatement A function that creeates the SQL statement to
+   *                    execute from the current element. Any DML or
+   *                    DDL statement is acceptable.
+   */
+  def flow[T](
+      session: SlickSession,
+      toStatement: JBiFunction[T, Connection, PreparedStatement]
```

Comment:
Use `akka.japi.Function2` instead, so the `SQLException` can be thrown and fail the stream properly.

Reply:
✅ Done